pennylane/templates/embeddings/angle.py

```diff
@@ -16,40 +16,14 @@
 """
 # pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
 import pennylane as qml
-from pennylane.templates.decorator import template
-from pennylane.templates import broadcast
-from pennylane.wires import Wires
+from pennylane.ops import RX, RY, RZ
+from pennylane.operation import Operation, AnyWires
 
 
-def _preprocess(features, wires):
-    """Validate and pre-process inputs as follows:
+ROT = {"X": RX, "Y": RY, "Z": RZ}
```

Comment:
Using a dict rather than the previous three if statements to check which gate is used.